packages/platform-test-suite/lib/test/getDAPISeeds.js

```diff
@@ -2,12 +2,14 @@ function getDAPISeeds() {
   return process.env.DAPI_SEED
     .split(',')
     .map((seed) => {
-      const [host, httpPort, grpcPort] = seed.split(':');
+      const [host, httpPort, grpcPort, ssl] = seed.split(':');
```

Comment:
We can use `DAPIAddress` to parse the string. `seeds` option accepts `DAPIAddress` objects as well.